code/game/machinery/computer/card.dm

```diff
@@ -264,6 +264,15 @@
 						//let custom jobs function as an impromptu alt title, mainly for sechuds
 						if(temp_t && modify)
 							modify.assignment = temp_t
+					if (!(temp_t in all_jobs_txt))
+						var/new_dept = input("Choose the departement this job belongs to.") as null|anything in departement_list
+						if (new_dept)
+							for (var/list/L in list(data_core.general, data_core.medical, data_core.security,data_core.locked))
+								if (L)
```

Comment:
I copypasted it from another proc.
This shouldn't ever be false, so it's probably better to put `if (L.len)`